doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc

```diff
@@ -0,0 +1,57 @@
+[id='configuring-image-registry-images-proc_{context}']
+= Configuring the image registry to use specific images
+
+During the basic configuration of the environment you can configure the custom registry to use specific images. For more information about configuring the basic environment, see <<operator-deploy-basic-proc_{context}>>.
+
+.Prerequisites
+* You started to configure a {PRODUCT} environment using {operator_name} and accessed the installer wizard according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+* You have set the URL of the image registry in the *Image registry* field under the *Custom registry* of the {PRODUCT} environment configuration according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+
+.Procedure
+
+* During the {PRODUCT} environment configuration, to use a specific image from the custom registry, perform one of the following tasks:
```

Comment:
I'd use "complete one of the following steps:"

Reply:
Done